packages/office-ui-fabric-react/src/components/Autofill/Autofill.tsx

```diff
@@ -75,9 +75,15 @@ export class Autofill extends BaseComponent<IAutofillProps, IAutofillState> impl
 
   public componentDidUpdate() {
     const value = this._value;
-    const { suggestedDisplayValue, shouldSelectFullInputValueInComponentDidUpdate } = this.props;
+    const { suggestedDisplayValue, shouldSelectFullInputValueInComponentDidUpdate, shouldSelectValueInComponentDidUpdate } = this.props;
     let differenceIndex = 0;
 
+    // We need to explicitly not select the text in the autofill if we are no longer focused. In IE11, selecting
+    // a input will also focus the input, causing other element's focus to be stolen.
```

Comment:
This comment doesn't really make sense in this context because this code has nothing to do with being focused or not at this point...

Reply:
Makes sense, I think it'll make more sense if I add it in ComboBox